release-notes/major10/10.8.0/valtimo-frontend-libraries.md

```diff
@@ -4,16 +4,9 @@
 
 The following features were added:
 
-* **New feature1**
-
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
-
-* **New feature2**
-
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
+* **IDs added to menu items (optional) and search field buttons**
 
+  The menu items can be configured to have an id just add the property in the enviorment file. The case search fields have ids added to the drop down arrow and the clear/search buttons.
```

Comment:
This does not fit the quality standards of the documentation. Unclear explanation, missing punctuation (... just add), spelling mistakes (enviorment). Please rewrite so that it is clear what has been changed, how exactly this can be used (i.e. what has to be changed in the environment file exactly etc.)